pkg/apis/kops/cluster.go

```diff
@@ -316,6 +316,16 @@ type EtcdClusterSpec struct {
 	HeartbeatInterval *metav1.Duration `json:"heartbeatInterval,omitempty"`
 	// Image is the etcd docker image to use. Setting this will ignore the Version specified.
 	Image string `json:"image,omitempty"`
+	// Backups describes how we do backups of etcd
+	Backups *EtcdBackupSpec `json:"backups,omitempty"`
```

Comment:
Nit pick ... your choice, but I do like `EtcdBackups` better thanks `Backups`.  Backups sound like we are backing up disk, really is too broad in my mind.  

Thoughts?

Reply:
I was just reading through this and thinking the same thing.  I see Backups is a `*EtcdBackupSpec` but that wouldn't be as apparent in an editor when modifying a cluster.

Reply:
So it looks like this now:

```
  etcdClusters:
  - backups:
      backupStore: s3://bucket/cluster.example.com/backups/etcd/main/
    etcdMembers:
    - instanceGroup: master-us-west-2a
      name: a
    name: main
  - backups:
      backupStore: s3://bucket/cluster.example.com/backups/etcd/main/
    etcdMembers:
    - instanceGroup: master-us-west-2a
      name: a
    name: events
```

And you're proposing it looks like this:
```
  etcdClusters:
  - etcdBackups:
      backupStore: s3://bucket/cluster.example.com/backups/etcd/main/
    etcdMembers:
    - instanceGroup: master-us-west-2a
      name: a
    name: main
  - etcdBackups:
      backupStore: s3://bucket/cluster.example.com/backups/etcd/main/
    etcdMembers:
    - instanceGroup: master-us-west-2a
      name: a
    name: events
```

I guess that's OK with the precedent of `etcdMembers`.  I don't love the repetition, but I guess it's fine...

Reply:
Actually seeing  the entire spec you are good.

Seeing it make sense

Reply:
I would have seen this better with a test or docs btw. I understand this is alpha, so we are not documenting it quite yet. But if you could do a test in a follow-up PR ;)

Reply:
Oh derp, that makes sense as just `backups`.